config/config.go

```diff
@@ -1,12 +1,16 @@
 package config
 
 import (
+	"fmt"
 	"gopkg.in/yaml.v2"
 	"io/ioutil"
 	"log"
 	"os"
+	"path"
 )
 
+var defaultHome = os.ExpandEnv("$HOME/.habbgo")
```

Comment:
https://github.com/adrg/xdg
https://pkg.go.dev/github.com/adrg/xdg#ConfigFile

Reply:
Ohh snap! This is awesome, gonna throw it in here right now.

Reply:
went ahead and added it in. this was a great addition, thanks for the tip!